poprox-db/migrations/versions/2025_11_05_0756-27d1fd295797_create_article_packages_tables.py

```diff
@@ -0,0 +1,96 @@
+"""create article packages tables
+
+Revision ID: 27d1fd295797
+Revises: 95080bdb0b0d
+Create Date: 2025-11-05 07:56:46.208063
+
+"""
+from typing import Sequence, Union
+
+import sqlalchemy as sa
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision: str = '27d1fd295797'
+down_revision: Union[str, None] = '95080bdb0b0d'
+branch_labels: Union[str, Sequence[str], None] = None
+depends_on: Union[str, Sequence[str], None] = None
+
+
+def upgrade() -> None:
+    op.create_table(
+        "packages",
```

Comment:
Are we worried about using too-general of a term for this? I don't know that I love `article_packages` for the table name, but `packages` sounds vague out-of-context.